_posts/2016-06-01-welcome-to-magenta.markdown

```diff
@@ -84,7 +84,7 @@ neural networks, such as
 [DeepDream](http://googleresearch.blogspot.com/2015/06/inceptionism-going-deeper-into-neural.html)
 from A. Mordvintsev et al. at Google and [Neural Style
 Transfer](http://arxiv.org/abs/1508.06576) from L. Gatys et al. at
-U. Tübingen. We believe this area is in its infancy, and expect to see
+U. Tübingen. We believe this area is in its infancy and expect to see
```

Comment:
Adding a comma here is grammatically wrong, I believe. It's not followed by an independent clause.